packages/mask/src/plugins/Trader/trader/uniswap/useTradeCallback.ts

```diff
@@ -123,11 +126,11 @@ export function useTradeCallback(
                     data: calldata,
                     ...('gasEstimate' in bestCallOption ? { gas: bestCallOption.gasEstimate.toFixed() } : {}),
                     ...(!value || /^0x0*$/.test(value) ? {} : { value }),
-                    ...gasConfig,
+                    ...pickBy(gasConfig, identity),
                 },
                 {
                     chainId,
-                    overrides: { ...gasConfig },
+                    overrides: { ...pickBy(gasConfig, identity) },
```

Comment:
Do we also remove empty fields in the `sendTransaction`?

```suggestion
                    overrides: { ...gasConfig },
```